main/FatesInterfaceMod.F90

```diff
@@ -1372,6 +1380,13 @@ subroutine set_fates_ctrlparms(tag,ival,rval,cval)
             call endrun(msg=errMsg(sourcefile, __LINE__))
          end if
 
+         if(hlm_parteh_model .eq. unset_int) then
+            if (fates_global_verbose()) then
+               write(fates_log(), *) 'switch deciding which plant reactive transport model to use'
```

Comment:
should this say 
'switch deciding which plant reactive transport model to use unset', as per the vertsoilc statement below?